app/presenters/content_item_presenter.rb

```diff
@@ -18,6 +18,12 @@ def available_translations
     sorted_locales(@content_item["links"]["available_translations"])
   end
 
+  def parent
+    if content_item["links"].include?("parent")
+      content_item["links"]["parent"][0]
```

Comment:
Should this method be provided by `Breadcrumb`? Both of these work together and have the same logic for extracting the first parent.